extensions/memorycache/memorycache.js

```diff
@@ -149,7 +153,8 @@ Loader.OnLoad(function() {
         address = new PointerMath.Pointer(address).value();
 
         var viewer = getArrayViewer(size, isUnsigned, isFloat);
-        if (!loadPage(getPage(address), viewer, function(view) {
+        if (isUnaligned(address, size) ||
```

Comment:
Can we add a comment or two about this check and why it is needed? Both here and below.

Reply:
Done

Reply:
LGTM. Please get sign off from @petersalas as well.